src/explore-education-statistics-common/src/components/ExportButtonMenu.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useCallback, useEffect, useRef } from 'react';
+import downloadFile from '@common/utils/file/downloadFile';
+import { useCurrentPng, useGenerateImage } from './ChartsToPNG';
+import ButtonText from './ButtonText';
+import styles from './ExportButtonMenu.module.scss';
+
+const ExportButtonMenu = () => {
```

Comment:
This is very specific to charts so should probably be in `charts/components` rather than the generic components folder. A more specific name would be better as well, something like `ChartExportButton`.